### ISSUE TITLE
mkdocs yaml file structure

### ISSUE THREAD

**Issue by amichuda** [minimod-nutrition/minimod-website#13]
@britsanto 

Just took a look at your pull request and things looked good, but I found that the site wasn't being created because of something in `mkdocs.yml`. This was due to a quirk in the way `yaml` works. So basically, if you wanted to have a page with a particular name, you can write:

```
- My Page: my_page.md
```

If you want a page that has sub-pages, you can do:

```
- My Page:
  - first page: page_1.md
  - second page: page_2.md
```

But you can't do both at the same time, which is what i found in your yaml file:

``` 
- My Page: my_page.md
   - first page: page_1.md
   - second page: page_2.md
```

**Comment by britsanto** [comment]
Oh, for the 'Tools' section Dr. Vosti had said that he wanted an overview for each tool so I thought that the format I used would work. Would it be better to just add an 'Overview' sub-page for each tool?

**Comment by amichuda** [comment]
> Oh, for the 'Tools' section Dr. Vosti had said that he wanted an overview for each tool so I thought that the format I used would work. Would it be better to just add an 'Overview' sub-page for each tool?

Yes, I think that would be perfect. I changed the `mkdocs.yml` file a little bit so make sure to pull new changes (as always, but just reminding you here)
